website/docs/index.html.markdown

```diff
@@ -186,7 +209,17 @@ The following arguments are used to configure the VMware vCloud Director Provide
    instead of username and password. When this is set, username and password will
     be ignored, but should be left in configuration either empty or with any custom values.
     A token can be specified with the `VCD_TOKEN` environment variable.
-    
+
+* `use_saml_adfs` - (Optional) Set `true` to use SAML login flow with Active Directory Federation
```

Comment:
In discussions about this field. Options so far:
`saml_adfs_enabled`
`saml_adfs_auth_enabled`
`auth_type = "saml_adfs" | "native" (default) | "token" | "ldap"` 

CC: @dataclouder @vbauzysvmware

Reply:
my vote auth_type if possible